config/symbols.hd.dra.txt

```diff
@@ -41,6 +41,7 @@ D_800ADC10 = 0x800ADCD4;
 D_800ADC44 = 0x800ADD08;
 D_800ADC7C = 0x800ADD40;
 D_800ADCC8 = 0x800ADD8C;
+D_800ADDE8 = 0x800ADEAC;
```

Comment:
I'm not sure why this is needed when it's already been extracted to `src/dra/d_DBD4.c` below? but the build was [failing](https://github.com/Xeeynamo/sotn-decomp/actions/runs/9822606885/job/27119583447) without it...

Reply:
Doesn't look like it's extracted in the HD version https://github.com/Xeeynamo/sotn-decomp/blob/master/config/splat.hd.dra.yaml

Reply:
Good catch